datahub-web-react/src/app/searchV2/autoCompleteV2/components/icon/DefaultEntityIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+import { EntityIconProps } from './types';
+import { SingleEntityIcon } from './SingleEntityIcon';
+
+const Container = styled.div`
+    display: flex;
+`;
+
+const ICON_SIZE = 20;
+const SIBLING_ICON_SIZE = 16;
+
+export default function DefaultEntityIcon({ entity, siblings }: EntityIconProps) {
+    const hasSiblings = useMemo(() => (siblings?.length ?? 0) > 0, [siblings?.length]);
+    const entitiesToShowIcons = useMemo(() => (hasSiblings ? siblings : [entity]), [hasSiblings, siblings, entity]);
```

Comment:
so typically i would imaging `siblings` to be a list of entities that aren't the main `entity` itself. also there's a chance there could be many siblings entities from the same platform (not typical, but some folks do do this) so we need to be careful we only show unique platforms.

so we should think about this more along the lines of what platforms we need to show than getting all the siblings. passing in the siblings entities makes enough sense then we just need to get all the platforms we want to render basically.